### PR TITLE
Fix notebook errors

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/222

Run mysql and paddlecloud in different pods.
Fix https://github.com/PaddlePaddle/cloud/issues/224

### CODE REVIEW

File: k8s/mysql_deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        hostPath:
+          path: /home/pcloud/mysql
+      containers:
+      - name: mysql
+        image: mysql:5.7
+        volumeMounts:
+        - name: mysql-data
+          mountPath: /var/lib/mysql
+        env:
+          - name: MYSQL_ROOT_PASSWORD
```
Comment (gongweibao) [inline k8s/mysql_deployment.yaml]: 需要加上:https://github.com/PaddlePaddle/cloud/blob/develop/k8s/cloud_deployment.yaml#L82
```
- name: MYSQL_DATABASE
    value:paddlecloud
```

File: k8s/cloud_deployment.yaml [hunk 2]
```diff
@@ -71,13 +71,5 @@ spec:
...
-        image: mysql:5.7
-        volumeMounts:
-        - name: mysql-data
-          mountPath: /var/lib/mysql
-        env:
-          - name: MYSQL_ROOT_PASSWORD
-            value: root
       nodeSelector:
-        kubernetes.io/hostname: k8s-node1
+        kubernetes.io/hostname: your-host-name
```
Comment (gongweibao) [inline k8s/cloud_deployment.yaml]: 这个我不是很清楚，这个的值在哪里设置？

File: paddlecloud/notebook/templates/notebook.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
   </script>
   {% if notebook_status == "running" %}
   <div class="container fullheight">
-    <iframe src="http://cloud.paddlepaddle.org/notebook/{{ notebook_id }}" width="100%" height="500" style="border:none;"></iframe>
+    <iframe src="/notebook/{{ notebook_id }}" width="100%" height="500" style="border:none;"></iframe>
```
Comment (gongweibao) [inline paddlecloud/notebook/templates/notebook.html]: 赞！

File: k8s/cloud_deployment.yaml (hunk 2 above)
Comment (typhoonzero) [inline k8s/cloud_deployment.yaml]: 根据集群配置节点。不是固定的值

File: k8s/mysql_deployment.yaml (hunk 1 above)
Comment (typhoonzero) [inline k8s/mysql_deployment.yaml]: Done.

**Pull request merged**